src/CppUTest/CommandLineArguments.cpp

```diff
@@ -149,6 +157,26 @@ void CommandLineArguments::SetRepeatCount(int ac, const char *const *av, int& i)
 
 }
 
+bool CommandLineArguments::SetShuffle(int ac, const char * const *av, int& i)
+{
+    shuffle_ = SHUFFLE_ENABLED_RANDOM_SEED;
+    SimpleString shuffleParameter(av[i]);
+    if (shuffleParameter.size() > 2) {
+        shuffle_ = SimpleString::AtoU(av[i] + 2);
+        if (shuffle_ == SHUFFLE_DISABLED) {
+            return false;
+        }
+    } else if (i + 1 < ac) {
+        const unsigned parsed = SimpleString::AtoU(av[i + 1]);
+        if (parsed != 0)
+        {
```

Comment:
I'll merge it, but would recommend to try to keep your style consistent :)

Reply:
Oops, missed that one. Thanks for merging!